Updating securing apps and authorization documentation based on the adapter deprecation/removal plan (#21177)
keycloak/keycloak: docs/documentation/securing_apps/topics/oidc/java/java-adapters.adoc

=== {project_name} Java adapters

[WARNING]
====

> {project_name} Java Adapters are now in End-Of-Life (EOL), and it is no longer supported nor receiving security updates.

Reviewer: ```suggestion
{project_name} Java Adapters are now in End-Of-Life (EOL) status. They are no longer supported nor receiving security updates.
```

> {project_name} Java Adapters are now in End-Of-Life (EOL) status. They are no longer supported nor receiving security updates.
> It will be removed in future releases.

Reviewer 2: This statement is probably too strong. We haven't EOL them quite yet, but rather deprecated. We'll do security updates on adapters until they are fully EOL, but won't do any further development.

Perhaps something like:
```suggestion
{project_name} Java Adapters are now deprecated and will be removed in a future release. No further enhancements or features will be added to the adapter until then.
```
